selfdrive/car/hyundai/values.py

```diff
@@ -66,6 +66,7 @@ class CAR:
   KIA_SORENTO = "KIA SORENTO GT LINE 2018"
   KIA_STINGER = "KIA STINGER GT2 2018"
   KIA_CEED = "KIA CEED INTRO ED 2019"
+  KIA_MORNING = "KIA ALL NEW MORNING 2019"
```

Comment:
this car seems to have many different regional names, I propose referring to it as the `Kia Picanto Morning` in openpilot. 

https://en.wikipedia.org/wiki/Kia_Picanto

Reply:
Thank you so much.
I hadn't thought of that part. I'll fix it as you said.